src/CardList/index.js

```diff
@@ -384,11 +383,11 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
   },
   rightMedia: {
-    height: 80,
-    width: 80,
-    marginTop: 4,
-    marginBottom: 8,
+    height: 100,
+    width: 100,
+    marginBottom: -25,
```

Comment:
What was the motivation for this change?

Reply:
Earlier when it was set to previous CSS values it was resulting in the image not showing on the right. So I added those CSS changes. This was before you pointed out the change:   style={[styles.image, imageStyles]}
I can revert back to the old CSS properties and check if just the change you proposed was enough to show the images on the right.